From 87f19e0f27d24f2cd30e6a66529d70e70b8bf310 Mon Sep 17 00:00:00 2001
From: Miquel Raynal <miquel.raynal@bootlin.com>
Date: Thu, 18 Oct 2018 15:19:06 +0200
Subject: [PATCH 0718/1051] ARM64: dts: a3900-db: remove legacy property

NAND timings should not come from the bootloader anymore. It just
hides problems and limits a lot the NAND throughput. Remove this
useless and legacy property.

Change-Id: I5af2bec35309eff31a4c8f2c5eb392f774569029
Signed-off-by: Miquel Raynal <miquel.raynal@bootlin.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/60387
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-3900-db.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
index c56943eb9e87..d08b07106594 100644
--- a/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-3900-db.dtsi
@@ -100,7 +100,6 @@
 		label = "main-storage";
 		nand-rb = <0>;
 		nand-ecc-mode = "hw";
-		marvell,nand-keep-config;
 		nand-on-flash-bbt;
 		nand-ecc-strength = <8>;
 		nand-ecc-step-size = <512>;
-- 
2.17.1

